pandas/tests/base/utils.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+
+import numpy as np
+
```

Comment:
rather than creating this file can you put in pandas/utils/testing.py (you may need to expand the names to conform a bit more)